### ISSUE TITLE
Update package baseline once 3.0 ships

### ISSUE THREAD

**Issue by JunTaoLuo** [aspnet/AspNetCore#12702]
As part of the branding update, I've chosen to maintained our 2.2 package baselines and continue to suppress dependency warnings. We should update the baselines instead once 3.0 ships.

cc @dougbu @anurse @Pilchie

**Comment by dougbu** [comment]
@wtgodbe also please remove `@(SuppressBaselineReference)` settings and any MSBuild blocks that _only_ contain those settings.

**Comment by dougbu** [comment]
@wtgodbe to clarify: The target branch for this is 'release/3.0' and it needs to be done in Extensions and AspNetcore

**Comment by wtgodbe** [comment]
I have a first pass up at https://github.com/aspnet/AspNetCore/commit/c1287d584b9eee8336944ab229bbebc3291f3677 - the first section is all of the packages, the second section is all projects from the shared framework. The `BaselineGenerator` project fails because it's trying to download packages for each element in the .xml file, when some of them are projects - should I just remove them when running the tool, or change the metadata for them from `package` to `project`? Also, the versions for the `Blazor` packages aren't stable - is that expected/acceptable? Finally, the last set of refs from the Shared Framework appear to be redistributed assemblies - https://github.com/aspnet/AspNetCore/commit/c1287d584b9eee8336944ab229bbebc3291f3677#diff-c763c910a1518ee2bfcbe97d4e86a6bcR212-R219. Should I remove these? @dougbu @JunTaoLuo @anurse please advise

**Comment by dougbu** [comment]
1. Only list packages
2. But, include some analogue of a shared framework project, perhaps [Microsoft.AspNetCore.App.Runtime.win-x64](https://www.nuget.org/packages/Microsoft.AspNetCore.App.Runtime.win-x64/)
  - not sure we have any Linux or macOS-specific product packages but if we do have such, include the corresponding shared framework packages
3. It's intended that Blazor versions are not stable in 3.0.0 GA. That's also true for the WASM (Web assembly aka client-side Blazor) packages in AspNetCore. They'll stabilize in 3.1.0.
4. We redistributed CoreFx assemblies in our shared framework in 2.x tool Do what we did then.
